src/shogun/base/init.h

```diff
@@ -108,6 +109,18 @@ void set_global_rand(CRandom* rand);
  */
 CRandom* get_global_rand();
 
+/** set the global linalg library object
+ *
+ * @param linalg linalg object to use
+ */
+void set_global_linlg(SGLinalg* linalg);
```

Comment:
Is it defined somewhere? I see definition of get_global_linalg() in the cpp but not this one. 

Also, maybe this is not needed. Just remove this for now :)